lib/count-closed-tickets-by-year.rb

```diff
@@ -1,29 +1,22 @@
 require 'zendesk_api'
 
-@client = ZendeskAPI::Client.new do |config|
-  config.url = ENV['ZENDESK_URL']
-  config.username = ENV['ZENDESK_USER_EMAIL']
-  config.password = ENV['ZENDESK_USER_PASSWORD']
+require_relative 'zendesk-setup.rb'
 
-  config.retry = true
-end
-
-today = Date.today.prev_day
-lastyear = today - 365
+lastyear = Date.today.prev_day - 365
 
 puts "Total Tickets: #{@client.tickets.count}"
 
 puts "Total Tickets 2012"
```

Comment:
Can these lines be replaced with the following code? 

```
current_year = Time.now.year
year = 2012
until year == current_year do
  puts "Total Tickets #{year}"
  if lastyear.year == year then
    puts @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>#{year}-01-01 updated_at<#{lastyear}").count.to_i
  else
    puts @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>#{year}-01-01 updated_at<#{year+1}-01-01").count.to_i
  end
  year += 1
end
```

Reply:
I have replaced `status:closed updated_at>#{year}-01-01` with `status:closed updated_at>=#{year}-01-01`. It needs to include 1st January.

Reply:
Resolved as suggested.